### DISCUSSION TITLE
freshrss/freshrss:arm removed?

### DISCUSSION THREAD

**Discussion by ghost** [FreshRSS/FreshRSS#5852] [Q&A] [answered]
Might be related to #5832, but arm tag seems to be missing.
```
pi@raspberry:~# docker pull freshrss/freshrss:arm
Error response from daemon: manifest for freshrss/freshrss:arm not found: manifest unknown: manifest unknown
```

**Comment by Alkarex** [comment, marked as answer]
Yes, we have transitioned to multi-architecture images. Please try `freshrss/freshrss:edge` or `freshrss/freshrss:1.21.1-arm`.

`freshrss/freshrss` will work again for ARM from next release

**Reply by Alkarex** [reply]
We have increased speed to release [FreshRSS 1.23.0](https://github.com/FreshRSS/FreshRSS/milestones/1.23.0) some time in December 🎄

**Reply by Alkarex** [reply]
https://github.com/FreshRSS/FreshRSS/discussions/5963

**Reply by Alkarex** [reply]
https://github.com/FreshRSS/FreshRSS/releases/tag/1.23.0
